ghost/core/core/server/services/email-analytics/lib/event-processor.js

```diff
@@ -135,7 +135,21 @@ class GhostEventProcessor extends EventProcessor {
     }
 
     async handleComplained(event) {
-        return this._unsubscribeFromNewsletters(event);
+        const emailId = await this.getEmailId(event);
+        let updateComplaint = false;
+
+        if (emailId) {
```

Comment:
@allouis , do we need to hide it behind feature flag? Or it's okay to leave it as is? So, we'll collect info, but won't show it in UI.